contracts/bonding/BondingManager.sol

```diff
@@ -319,7 +327,6 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
         external
         whenSystemNotPaused
         currentRoundInitialized
-        autoClaimTokenPoolsShares
```

Comment:
Currently, you cannot withdraw your bonded stake unless you're in the unbonded state and you can only be in the unbonded state if you previously called `unbond()` and waited through the unbonding period without bonding again. As a result, you wouldn't have any earnings that you need to claim if you're calling `unbond()` since you would not have been delegated to a transcoder during the unbonding period.